x-pack/plugins/maps/public/components/tooltip_selector/add_tooltip_field_popover.test.tsx

```diff
@@ -18,15 +18,17 @@ const defaultProps = {
     },
     {
       name: 'prop2',
+      label: 'prop2',
```

Comment:
if we're adding this to test,  I'd give the label a different name. `prop2-label` and `@timestamp-label`